README.rst

```diff
@@ -86,7 +86,26 @@ step name.
 
 .. note:: Artifacts from previous steps are not available within remote builds
 
-Jinja Templating
+There are two image builders in ``buildrunner``. The default builder is
+equivalent to ``docker build`` and only supports single-platform images,
+this is reference as the legacy image builder.
+The new image builder is equivalent to ``docker buildx`` and is used for for both
+single and multi-platform images. To use the ``docker buildx`` builder,
+set ``use-legacy-builder: false`` in the configuration file or use ``platforms``
+in the ``build`` section. Eventually the legacy builder will be removed.
```

Comment:
`The legacy build will be removed in a future release.` "Eventually" could be a very long time away, and it's really not.

Reply:
I changed it to `The legacy builder will be removed in a future release.`

Reply:
I squashed these changes into the previous commit.